src/sass/components/_case-study-teaser.scss

```diff
@@ -3,6 +3,14 @@
   display: inline-flex;
   justify-content: center;
   align-items: center;
+  .img-overlay {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    transition: 0.5s ease;
+    background-color: $colors-grey-90;
+  }
   &__info {
     max-height: 100px;
```

Comment:
Can we convert all px values to rems? There are a sass functions that do that.
eg this is from the uswds utilities

```
@function px-to-rem($pixels) {
  @if not $pixels {
    @return false;
  }
  $px-to-rem: ($pixels / $root-font-size-equiv) * 1rem;
  $px-to-rem: round($px-to-rem * 100) / 100;

  @return $px-to-rem;
}
```

Reply:
@iris-i Should this be a separate pr for the whole codebase? What would u recommend for a `root-font-size-equiv` ?

Reply:
@Yodart I believe 16px is the common base font-size converting px to rem. Also, sounds  fine to do this in a separate ticket/PR.